app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt

```diff
@@ -0,0 +1,97 @@
+package org.oppia.app.testing
+
+import android.view.View
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.intent.Intents
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.FirebaseApp
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.ongoingtopiclist.OngoingTopicListActivity
+import org.oppia.app.ongoingtopiclist.OngoingTopicListFragment
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+class OngoingTopicListSpanTest {
+
+  @Before
+  @ExperimentalCoroutinesApi
+  fun setUp() {
+    Intents.init()
+    FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext())
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private fun getOngoingRecyclerViewGridLayoutManager(
+    activity: OngoingTopicListActivity
+  ): GridLayoutManager {
+    return getOngoingRecyclerView(activity).layoutManager as GridLayoutManager
+  }
+
+  private fun getOngoingRecyclerView(activity: OngoingTopicListActivity): RecyclerView {
+    return getOngoingTopicListFragment(activity).view?.findViewWithTag<View>(
+      activity.resources.getString(R.string.ongoing_recycler_view_tag)
+    )!! as RecyclerView
+  }
+
+  private fun getOngoingTopicListFragment(
+    activity: OngoingTopicListActivity
+  ): OngoingTopicListFragment {
+    return activity
+      .supportFragmentManager
+      .findFragmentByTag(
+        OngoingTopicListFragment.TAG_ONGOING_TOPIC_LIST_FRAGMENT
+      ) as OngoingTopicListFragment
+  }
+
+  @Test
+  fun testOngoingTopicListFragmentRecyclerView_hasCorrectSpanCount() {
```

Comment:
Rename to `testOngoingTopicList_checkRecyclerView_hasCorrectSpanCount`

Ditto for other tests.

Reply:
done